docs/sdkx_android/tracking.mdx

```diff
@@ -122,75 +133,69 @@ If you want to set Custom Issue Fields at the time of Issue creation, follow the
     Helpshift.showConversation(MainActivity.this, config);
 ```
 
-The following are the valid values for the `type` key of a Custom Issue Field.
-
-- "singleline"
-- "multiline"
-- "number"
-- "checkbox"
-- "dropdown"
-- "date"
-
-Compatibility table for type and values:
+Possible datatypes to be passed into the config are:
 
-| Type       | Value   | Comments                                                        |
-| ---------- | ------- | --------------------------------------------------------------- |
-| singleline | string  | Character limit of 255                                          |
-| multiline  | string  | Character limit of 100,000                                      |
-| number     | string  |                                                                 |
-| dropdown   | string  | Drop-down options should exist for the given Custom Issue Field |
-| date       | number  | Epoch time. For example - **Date.now()**                        |
-| checkbox   | boolean |                                                                 |
+| Type         | value         | Comments                                                                                                                                                      |
+| ------------ | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| "singleline" | string        | Single line string with character limit of 255                                                                                                                |
+| "multiline"  | string        | Multi line string with character limit of 100,000                                                                                                             |
+| "number"     | string        | String representation of number. For eg. "12345"                                                                                                              |
+| "dropdown"   | string        | Drop-down options should exist for the given Custom Issue Field on dashboard. Value should be one of the values specified on the dashbaord for that dropdown. |
+| "date"       | long long int | Epoch time in milliseconds. For eg. `1505927361535`                                                                                                           |
```

Comment:
"long long int", should be just long right ?

Reply:
Right. Fixed.